code/modules/cargo/delivery/package.dm

```diff
@@ -0,0 +1,86 @@
+/obj/item/cargo_package
+	name = "cargo package"
+	desc = "An Orion Express cargo package, these packages generally find their way to researchers bunkered up on exoplanets. Always pays an extra 2% tip to the courier."
+	desc_info = "You can deliver this package to a delivery site on an exoplanet to get additional funds for the cargo department's account. An additional 2% is added to your account on delivery. Can be loaded into a cargo pack."
+	icon = 'icons/obj/orion_delivery.dmi'
+	icon_state = "express_package"
+	item_state = "express_package"
+	contained_sprite = TRUE
+	w_class = ITEMSIZE_HUGE
+	force = 10
+
+	slowdown = 1
+
+	var/delivery_point_id = ""
+	var/delivery_point_sector = ""
+	var/delivery_point_coordinates = ""
+
+	var/datum/weakref/associated_delivery_point
+	var/pay_amount = 69420
+
+/obj/item/cargo_package/Initialize(mapload, obj/structure/cargo_receptacle/delivery_point)
+	. = ..()
+	pay_amount = rand(4, 7) * 1000
+	if(prob(3))
+		pay_amount = rand(12, 17) * 1000
+	if(delivery_point)
+		associated_delivery_point = WEAKREF(delivery_point)
+		delivery_point_id = delivery_point.delivery_id
+		delivery_point_sector = delivery_point.delivery_sector
+		delivery_point_coordinates = "[delivery_point.x]-[delivery_point.y]"
+
+/obj/item/cargo_package/examine(mob/user, distance)
+	. = ..()
+	to_chat(user, SPAN_NOTICE("The label on the package reads: SITE: <b>[delivery_point_sector]</b> | COORD: <b>[delivery_point_coordinates]</b> | ID: <b>[delivery_point_id]</b>"))
+	to_chat(user, SPAN_NOTICE("The price tag on the package reads: <b>[pay_amount]电</b>"))
+
+/obj/item/cargo_package/do_additional_pickup_checks(var/mob/living/carbon/human/user)
+	if(!ishuman(user))
+		to_chat(user, SPAN_WARNING("\The [src] is way too heavy for you to pick up!"))
```

Comment:
this would be a bit of a weird message for a borg to get, I'd probably just remove it